lib/sentry/client.ex

```diff
@@ -51,7 +51,7 @@ defmodule Sentry.Client do
   require Logger
 
   @type send_event_result ::
-          {:ok, Task.t() | String.t() | pid()} | {:error, any()} | :unsampled | :excluded
```

Comment:
I removed `{:ok, pid}` from `send_event_result` type.

Reply:
I can split it into a separate PR like "fix type send_event_result type in client.ex"

Reply:
Moved to https://github.com/getsentry/sentry-elixir/pull/545